src/Features/Core/Portable/QuickInfo/CommonSemanticQuickInfoProvider.cs

```diff
@@ -171,159 +169,9 @@ private static bool HasNoErrors(ImmutableArray<ISymbol> symbols)
                     s2.Kind == SymbolKind.NamedType ? 1 : 0).ToImmutableArray();
             }
 
-            var groups = await descriptionService.ToDescriptionGroupsAsync(workspace, semanticModel, token.SpanStart, symbols, cancellationToken).ConfigureAwait(false);
-
-            var sections = ImmutableArray.CreateBuilder<QuickInfoSection>(initialCapacity: groups.Count);
-
-            var symbol = tokenInformation.Symbols.First();
-            if (tokenInformation.ShowAwaitReturn)
```

Comment:
this logic was moved, effectively unchanged.  we just now don't access token/tokenInformation/etc. anymore, but we use the values passed in.